### ISSUE TITLE
Sort library categories differently

### ISSUE THREAD

**Issue by Ubuntu-2020** [kiwix/libkiwix#979]
I suggest changing the sorting by category.
Now the sorting is in the English alphabet: A-B-C - etc.:
![Library-Kiwix 2 0 1-1 appimage](https://user-images.githubusercontent.com/65112331/88167485-92f8fe00-cc21-11ea-9ea3-7c7332501c07.png)

I suggest changing this list to look like this:
- All
- Wikipedia
- Wikinews
- Wikiversity
- Wikivoyage
- Wikiquote
- Wikisource
- Wiktionary
- Vikidia
- Psiram
- Stack Exchange
- Other

PS: maybe you should delete **[Psiram](https://github.com/openzim/mwoffliner/issues/1216)**?

**Comment by mgautierfr** [comment]
This is not exactly the same problem.
Here it is easy to found a sorting order: The category name in the current ui language.
The problem with language is that it is the first thing a user will want to change if it is not its language (or a language they cannot read).
We can expect that the user will interact with the category selector in a language they understand.

**Comment by kelson42** [comment]
@mgautierfr Yes, so first we need tomtranslate the categories and then we need to sort them (like you said).
